### DISCUSSION TITLE
Wrong system shown in deployment view

### DISCUSSION THREAD

**Discussion by zhelyan** [structurizr/java#325] [Q&A]
### Question

A deployment view for the email system shows the mainframe system. Is this expected?

![1000007471](https://github.com/user-attachments/assets/b10d13c3-a2e3-4c2a-a244-6cab539a18c8)

**Comment by simonbrowndotje** [comment]
Yes, this is expected; see https://docs.structurizr.com/dsl/language#deployment-view ... for a software system scoped deployment view, `include *` includes:
- All deployment nodes and infrastructure nodes within the deployment environment <- this gives you Big Bank plc, bigbank-prod001, and the Mainframe Banking System
- Container instances within the deployment environment that belong to the software system <- there are no containers defined for the E-mail System
